tool/planet/agent.go

```diff
@@ -477,6 +503,47 @@ func status(c statusConfig) (ok bool, err error) {
 	return ok, nil
 }
 
+func validateKubernetesService(ctx context.Context, serviceCIDR net.IPNet) error {
+	return utils.Retry(ctx, math.MaxInt64, 5*time.Second, func() error {
+		client, err := cmd.GetKubeClientFromPath(constants.SchedulerConfigPath)
```

Comment:
Does this method run on regular nodes too? It is called from the leader callback which doesn't seem to have any filters to run only on masters. Asking b/c regular nodes don't have scheduler config.